config/control_ur5.yaml

```diff
@@ -0,0 +1,29 @@
+arm_controller:
```

Comment:
MoveIt becomes unresponsive. I'm not sure what else had to be changed, but it's not as simple as changing arm_controller -> husky_arm_controller.